dist/intern.d.ts

```diff
@@ -1,27 +1,23 @@
 export declare class intern {
     static asyncMethod(f: any): (this: any, msg: any, done: any, meta: any) => any;
-    static findKnex(ent: any, q: any): Promise<any>;
-    static firstKnex(ent: any, q: any): Promise<any>;
-    static insertKnex(ent: any, data: any): Promise<any>;
-    static updateKnex(ent: any, data: any): Promise<any>;
-    static removeKnex(ent: any, q: any): Promise<any>;
-    static upsertKnex(ent: any, data: any, q: any): Promise<any>;
+    static findKnex(ent: any, q: any, knex: any): Promise<any>;
```

Comment:
let's make this api more "predictable" - see https://wiki.c2.com/?PrincipleOfLeastAstonishment

let's use the arg order: 
`knex, ent, q`
moving from left to right, from most general, to most specific

Reply:
Done. Wonderful, I learned a new principle : ) .